NEWS.md

```diff
@@ -1,12 +1,29 @@
-Changes in 2.0.2:
+## Changes in 2.0.3:
```

Comment:
maybe double check here to make sure we are getting everything?

https://github.com/neurodata/RerF/compare/master...staging

Reply:
do we include our bugfixes here?  like the issue with splitting on an empty node?